Bottom: 97e4f88000b5fc065cccbf8ee26d6e3ee483792c
Top:    c8a0023fc872b952d35873d8c9f60bfc2ac5cd37
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-18 14:15:52 -0500

Refresh of all-docker_container

---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index f286d43..23f849e 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -109,7 +109,7 @@
     hostname: '{{ container_addr_map.exim4.hostname }}'
     networks:
       - name: '{{ docker_network_frontend }}'
-    ports: '{{ exim4_ports }}'
+    ports: '{{ container_port_map.exim4.port_args }}'
     image: '{{ exim4_image_name }}:{{ docker_image_tag }}'
     restart_policy: no
     state: started
